sdk/src/wallet/core/operations/get_account.rs

```diff
@@ -16,26 +16,30 @@ impl<S: SecretManage> Wallet<S> {
         identifier: I,
     ) -> crate::wallet::Result<Account<S>> {
         let account_id = identifier.into();
-        let accounts = self.accounts.read().await;
+
+        todo!("since there's only one account in the wallet, no need to have AccountIdentifier?");
```

Comment:
No need to have the whole function anymore, we might want something for account outputs later, but even if so, it shouldn't be done in this PR